src/transform-sdk/rust/core-types/src/lib.rs

```diff
@@ -116,13 +116,36 @@ impl<'a> From<&'a WrittenRecord<'a>> for BorrowedRecord<'a> {
     }
 }
 
+/// [`WriteOptions`] allows for customizing a [`RecordWriter`]'s write.
+///
+/// An example is to customize the output topic which a write can go to.
```

Comment:
```suggestion
/// For example, use [`WriteOptions`] to customize the output topic to write to.
```
Are there any behaviors or cases (other than the output topic) that WriteOptions can be used to customize?

Reply:
Not now, but in the future maybe